lib/push-jenkins-update.js

```diff
@@ -118,6 +132,25 @@ function createGhStatus (options, logger, cb) {
   })
 }
 
+function findExistingCiComment (options, logger, cb) {
+  async.parallel([
+    botUsername.resolve,
```

Comment:
This can be a constant, it's not going to change, and it's costing us and API call.

Reply:
True enough.

Finding the bot's username via the GitHub API was made with a good intention a while back, but I won't argue with you it surely smells like overengineering.

Reply:
https://github.com/nodejs/github-bot/pull/227